### DISCUSSION TITLE
Im so confused someone help me pls 😭

### DISCUSSION THREAD

**Discussion by Arthura0121** [3kh0/ext-remover#372] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I tried the versions 106 and up exploit because I have version 107 on my school chrome book so I did the instructions, I pressed esc as soon as I clicked view in chrome web store and did the bookmark and pasted the id of the goguardian extension in it but I'm not so sure if its working because its turning on and off over and over again and I don't know if its supposed to be like that...someone pls help meeeeee😭

**Comment by Aka-but-nice** [comment]
yes

**Comment by Aka-but-nice** [comment, marked as answer]
That is exactly what is supposed to be happening

**Reply by Aka-but-nice** [reply]
Also, if sites are still blocked, it might be because that method is a little old. Also, did it load into a blank page?

**Reply by Arthura0121** [reply]
yes it did load into about:blank after I pressed the bookmark. But for some reason it would work for like 2 minutes but then it would stop doing it so goguardian keeps on working for me :(

**Reply by Arthura0121** [reply]
also, since it works kinda, why does it have to turn on and off? can't it just be off when the code runs? isn't that easier?

**Reply by obkduishinijf** [reply]
> also, since it works kinda, why does it have to turn on and off? can't it just be off when the code runs? isn't that easier?

no, what the code does is it keeps constantly reloading the extension so it doesn't even have time to block anything. it can't just be forced off like that in v102+ (atleast without using the extension manifest + console method)

**Reply by Aka-but-nice** [reply]
Try the method after the method you used on the ext remover page

**Reply by Arthura0121** [reply]
ok

**Reply by Arthura0121** [reply]
I just tried but the tinyurl for goguardian doesn't work. when I open the website, it says, "Your file couldn’t be accessedIt may have been moved, edited, or deleted.
ERR_FILE_NOT_FOUND"
idk what to do now

**Reply by obkduishinijf** [reply]
> I just tried but the tinyurl for goguardian doesn't work. when I open the website, it says, "Your file couldn’t be accessedIt may have been moved, edited, or deleted. ERR_FILE_NOT_FOUND" idk what to do now

goguardian has a new blocking method, just wait for something new to come out

**Reply by Arthura0121** [reply]
alr

**Reply by Aka-but-nice** [reply]
U could try the chrome://signOut method. I'm sure someone here would help you!

**Reply by Arthura0121** [reply]
> U could try the chrome://signOut method. I'm sure someone here would help you!

How do you do that?

**Reply by Aka-but-nice** [reply]
> also, since it works kinda, why does it have to turn on and off? can't it just be off when the code runs? isn't that easier?

Don't you know anything about point-blank? What it does is that because of the Same-policy-origin when you open that blank page has the exact same permissions as extensions so if you mess with the file access API, which is this
![image](https://user-images.githubusercontent.com/119691881/213698358-9d2dc933-a1e9-48db-b519-93a0cec39dc7.png)
it reloads the extension over and over, it's as if you had a bot turning on and off file access to your blocker, neet ain't it?

**Reply by rwg100** [reply]
> > U could try the chrome://signOut method. I'm sure someone here would help you!
> 
> How do you do that?

i do that all the time bro idk why no one else does it

**Reply by CheezeKake77** [reply]
> i do that all the time bro idk why no one else does it

chrome://settings is blocked for many districts

**Reply by Aka-but-nice** [reply]
I have another link, by Bypassi told me not tell, if you have a discord I'll send the link. Now if the link I give you is blocked, try <a href="https://github.com/3kh0/ext-remover">New Point Blank</a>

**Reply by Arthura0121** [reply]
> I have another link, by Bypassi told me not tell, if you have a discord I'll send the link. Now if the link I give you is blocked, try [New Point Blank](https://github.com/3kh0/ext-remover)

alr, he can send it to my discord: I'mNotArthur#8542

**Reply by Arthura0121** [reply]
ty

**Comment by obkduishinijf** [comment]
that's what it's supposed to do lol

**Reply by Aka-but-nice** [reply]
also did you ever close that tab
